src/legacy/core_plugins/data/public/search/aggs/agg_config.ts

```diff
@@ -38,37 +36,9 @@ export interface AggConfigOptions {
   enabled?: boolean;
   id?: string;
   params?: Record<string, any>;
-  schema?: string | Schema;
+  schema?: string;
 }
 
-const unknownSchema: Schema = {
-  name: 'unknown',
-  title: 'Unknown', // only here for illustrative purposes
-  hideCustomLabel: true,
-  aggFilter: [],
-  min: 1,
-  max: 1,
-  params: [],
-  defaults: {},
-  editor: false,
-  group: AggGroupNames.Metrics,
-  aggSettings: {
-    top_hits: {
-      allowStrings: true,
-    },
-  },
-};
-
-const getSchemaFromRegistry = (schemas: any, schema: string): Schema => {
```

Comment:
vis.type.schemas.all were passed in to AggConfigs before, when setting the schema we would look it up in there and assign schema instance to AggConfig. N